app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonsLayout.kt

```diff
@@ -29,19 +31,27 @@ internal sealed interface DashboardLessonsEvent {
 @Composable
 @OptIn(ExperimentalFoundationApi::class)
 internal fun LessonsLayout(viewModel: LessonsViewModel = viewModel(), onEvent: (DashboardLessonsEvent) -> Unit = {}) {
-    val lessons by viewModel.lessons.collectAsState()
+    val lessons by viewModel.lessons.collectAsState(emptyList())
+    val selectedLanguage by viewModel.selectedLanguage.collectAsState(Language(Locale.getDefault()))
```

Comment:
the `Language(Locale.getDefault())` isn't good since that won't always be the appLanguage on older versions of Android.

If you make `viewModel.selectedLanguage` a `StateFlow` then you won't need to pass a parameter to `.collectAsState()`